vms/platformvm/api/static_service.go

```diff
@@ -100,13 +96,33 @@ type Owner struct {
 	Addresses []string    `json:"addresses"`
 }
 
+// APIL1Validator is the representation of a L1 validator sent over APIs.
+type APIL1Validator struct {
+	NodeID    ids.NodeID  `json:"nodeID"`
+	Weight    json.Uint64 `json:"weight"`
+	StartTime json.Uint64 `json:"startTime"`
+	BaseL1Validator
+}
+
+// BaseL1Validator is the representation of a base L1 validator without the common parts with a staker.
+type BaseL1Validator struct {
+	ValidationID *ids.ID `json:"validationID,omitempty"`
+	// PublicKey is the compressed BLS public key of the validator
+	PublicKey             *types.JSONByteSlice `json:"publicKey,omitempty"`
+	RemainingBalanceOwner *Owner               `json:"remainingBalanceOwner,omitempty"`
+	DeactivationOwner     *Owner               `json:"deactivationOwner,omitempty"`
+	MinNonce              *json.Uint64         `json:"minNonce,omitempty"`
+	// Balance is the remaining amount of AVAX this L1 validator has for paying
+	// the continuous fee, according to the last accepted state. If the
+	// validator is inactive, the balance will be 0.
+	Balance *json.Uint64 `json:"balance,omitempty"`
+}
+
 // PermissionlessValidator is the repr. of a permissionless validator sent over
 // APIs.
 type PermissionlessValidator struct {
 	Staker
-	// Deprecated: RewardOwner has been replaced by ValidationRewardOwner and
-	//             DelegationRewardOwner.
-	RewardOwner *Owner `json:"rewardOwner,omitempty"`
```

Comment:
Should this be kept around for retro-compatibility, even-though deprecated?

Reply:
It's been deprecated for 3 years, I think it was already long due that we should have removed it.